get-config.js

```diff
@@ -1,13 +1,17 @@
 
 module.exports = function getConfig() {
     const configPath = process.cwd() + '/src/config';
+    const nearEnv = process.env.NODE_ENV || process.env.NODE_ENV || 'development'
```

Comment:
two times process.env.NODE_ENV?

Reply:
oops, yeah, not tested yet – will do before pushing release